src/main/java/com/synopsys/integration/blackduck/service/dataservice/RoleService.java

```diff
@@ -51,7 +51,7 @@ private List<RoleView> getScopedRoles(BlackDuckRequestFilter scope) throws Integ
         return blackDuckApiClient.getAllResponses(requestMultiple);
     }
 
-    private BlackDuckRequestFilter createScopeFilter(String scope) {
+    public static BlackDuckRequestFilter createScopeFilter(String scope) {
```

Comment:
This allows us to create a GET parameter like `?filter=scope:server`. I thought I'll reuse this code instead of re-implementing it in the `UserGroupService`.